pandas/core/tools/datetimes.py

```diff
@@ -786,6 +786,10 @@ def to_datetime(
         return will have datetime.datetime type (or corresponding
         array/Series).
 
+        In case input is list-like and the elements of input are of mixed
```

Comment:
Mind adding this paragraph and an example in the `Examples` section

Reply:
Sure, I've added it with the `Examples` section.

Reply:
This paragraph can now be removed since you have it in the examples now

Reply:
sure, will remove it from the `Returns`